Tasks/DeployVisualStudioTestAgent/TestAgentInstall.ps1

```diff
@@ -34,6 +34,7 @@ function Install-Product($SetupPath, $UserName, $Password, $ProductVersion, $Arg
 		}
 		catch
 		{
+        	Write-Host "##vso[task.logissue type=error;code=001007;]"
```

Comment:
when i attempt to fix the indentation the client is not detecting it as a change